### PR TITLE
linspace -> LinRange

### PR DESCRIPTION
**Pull request by ghuba**
linspace -> LinRange
for Julia v1.0

**I have not tested this modification.**

### CODE REVIEW

File: src/convenience_api.jl [hunk 1]
```diff
@@ -69,7 +69,7 @@ keyword arguments are applied to the constructed trace.
 function Plot(f::Function, x0::Number, x1::Number, l::Layout=Layout();
               style::Style=CURRENT_STYLE[],
               kwargs...)
-    x = linspace(x0, x1, 50)
+    x = LinRange(x0, x1, 50)
```
Comment (Nosferican) [inline src/convenience_api.jl]: Shouldn't that just be
```
range(x0, stop = x1, length = 50)
```

Comment (ghuba) [reply src/convenience_api.jl]: @Nosferican That is correct too. But does that make a difference? `LinRange` is a type of range.

Comment (sglyon) [reply src/convenience_api.jl]: I think Either works, but I prefer the `range` construct as I think it is here to stay.

I'm not sure if LinRange will be considered internal at some point.

@ghuba would you be willing to make that change?

Thanks!

Comment (sglyon) [reply src/convenience_api.jl]: @ghuba looks like we posted at the same time.

If this works and is permanent, then I'm ok with with!

**Comment by ghuba** [comment]
I pushed the new change with range now.

**Pull request merged**